packages/shared/src/lib/flows/triggers/trigger.ts

```diff
@@ -37,7 +37,7 @@ export const BlockTriggerSettings = Type.Object({
   packageType: Type.Enum(PackageType),
   triggerName: Type.Optional(
     Type.String({
-      description: 'The name of the trigger should always be "trigger".',
+      description: 'The name of the trigger',
     }),
   ),
```

Comment:
Incorrect because this is actually the trigger action name (eg: triggerName: "every_week")